common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala

```diff
@@ -100,12 +100,12 @@ class CouchDbRestClient protected (system: ActorSystem, protocol: String, host:
         }
 
         futureBody.map { body =>
-            HttpRequest(method = method, uri = uri, headers = headers, entity = body)
+            HttpRequest(method=method, uri=uri, headers=headers, entity=body)
```

Comment:
I made the change (which really is a revert of another commit in this PR) with the intent of separating two concerns:

1. enforcing certain compiler-checkable properties
2. enforcing style rules

I think this PR should be about 1, and only 1. We cannot currently enforce style so I'd rather that we don't introduce all these changes.

The reason the PR altered the style in so many places is because @markusthoemmes apparently configures his editor to format on save. I'm happy to do the same but before we do that we need a way to do it consistently, and we need to agree on what the preferred style is.